File: HueQuickStartApps/HueQuickStartApp-iOS/HueQuickStartApp/Bridge/PHBridgeInfo.m
/*******************************************************************************
 Copyright (C) 2017 Philips Lighting Holding B.V.
 All Rights Reserved.
 ********************************************************************************/

#import "PHBridgeInfo.h"

@implementation PHBridgeInfo

+ (PHBridgeInfo *)createWithIpAddress:(NSString *)ipAddress andUniqueId:(NSString *)uniqueId {
    PHBridgeInfo* bridgeInfo = [PHBridgeInfo new];
    bridgeInfo.ipAddress = ipAddress;
    bridgeInfo.uniqueId = uniqueId;
    
    return bridgeInfo;
}

@end

File: HueQuickStartApps/HueQuickStartApp-iOS/HueQuickStartAppSwift/Bridge/PHBridgeInfo.swift
/*******************************************************************************
 Copyright (C) 2017 Philips Lighting Holding B.V.
 All Rights Reserved.
 ********************************************************************************/

import Foundation

struct PHBridgeInfo {
    let ipAddress:String
    let uniqueId:String
}

typealias PHBridgeInfoDiscoveryResult = PHBridgeInfo
extension PHBridgeInfoDiscoveryResult {
    init(withDiscoveryResult discoveryResult:PHSBridgeDiscoveryResult) {
        self.ipAddress = discoveryResult.ipAddress
        self.uniqueId = discoveryResult.uniqueId
    }
}
